user/pages/03.commerce2/02.developer-guide/02.install-update/07.patching/docs.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Patching
+taxonomy:
+    category: docs
+---
+
+Drupal uses a patch system to provide solutions to issues in between version
+releases. If you are unfamiliar with the concept of patching, you can learn
+about [Patches] at drupal.org.
+
+You can use Composer to apply patches by modifying the `composer.json` file
```

Comment:
This only works if they used composer create-project VS adding Commerce to a vanilla site. Should be clarified.